src/app/api/index.tsx

```diff
@@ -90,8 +90,8 @@ export const getNpmVersion = async (packageName: string): Promise<string | undef
 
 export const getRoadmap = async (): Promise<RoadmapBucket[] | undefined> => {
     try {
-        const response = await roadmap.get('/roadmap');
-        if (response && response.status === 200) return response.data;
+        const response = await roadmap.get('/roadmap.json');
```

Comment:
suggesting we version these by calling them something like `R16Roadmap.json`

Reply:
I can update the database and start versioning out our roadmaps.